tsconfig.base.json

```diff
@@ -18,6 +18,7 @@
     "baseUrl": ".",
     "strict": true,
     "paths": {
+      "*": ["libs/shared/types/src/lib/*", "*"],
```

Comment:
Why do you need this?
And shouldn't this be in the typeroots section?

Reply:
We spent some time with @arnifreyrs trying to debug that, moving this path as a `typeRoots` never worked. It was giving the following issue:

```text
Could not find a declaration file for module 'entropy-string'. 'node_modules/entropy-string/entropy-string.js' implicitly has an 'any' type.
  Try `npm install @types/entropy-string` if it exists or add a new declaration (.d.ts) file containing `declare module 'entropy-string';`ts(7016)
```

According to this [comment](https://stackoverflow.com/a/40326719), "`typeRoots` are NOT general-purpose ways to load arbitrary declaration (*.d.ts) files."

And it results using this path `*` that matches a library with a definition file if it exists in the directory `libs/shared/types/src/lib/*`.